src/cdaioctl.h

```diff
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: LGPL-2.1 */
```

Comment:
Once you're clear on which license you want, you should probably squash these two commits (`git rebase -i HEAD~3`).

Reply:
I have discussed the license LGPL-2.1 vs GPL-2.0 internally and we might go with GPL-2.0. Do you see any issues with it? I am waiting for inputs from one another person on this.

Reply:
I am still debating on dual license for this file. We shall make the change once we have consensus.

Reply:
We have decided to go with LGPL-2.1